packages/components/paginator/src/page.ts

```diff
@@ -0,0 +1,45 @@
+import { type CSSResultGroup, LitElement, type TemplateResult, html } from 'lit';
+import styles from './page.scss.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-page': Page;
+  }
+}
+
+/**
+ * A page component that can be used as part of the paginator, representing pages in the paginator.
+ *
+ */
+export class Page extends LitElement {
+  /** @internal */
+  static override styles: CSSResultGroup = styles;
+
+  /** @internal Page number used for the aria-label. */
+  #pageNumber = '';
```

Comment:
Why not do `@state() pageNumber?: string;`? That way, when `slotchange` event fires, it automatically rerenders the template.